packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -473,6 +488,13 @@ export class DetailsListBase extends BaseComponent<IDetailsListProps, IDetailsLi
     return <DetailsHeader {...detailsHeaderProps} />;
   };
 
+  private _onRenderDetailsFooter = (
+    detailsFooterProps: IDetailsFooterProps,
+    defaultRender?: IRenderFunction<IDetailsFooterProps>
+  ): JSX.Element => {
+    return <DetailsFooter {...detailsFooterProps} />;
+  };
```

Comment:
Default implementation should be `return null`. We want to avoid mounting another component if we won't need it.
There really shouldn't be a `DetailsFooter` provided in `office-ui-fabric-react` since we won't need one.